### DISCUSSION TITLE
How to switch location for the language servers?

### DISCUSSION THREAD

**Discussion by David-Else** [helix-editor/helix#2605] [Q&A] [answered]
I have started using the cool Neovim LSP installer https://github.com/williamboman/nvim-lsp-installer and now all my servers live at `~/.local/share/nvim/lsp_servers`. What is the best way to tell Helix they are there?

**Comment by kirawi** [comment]
I think you'll just have to add that directory to your `$PATH`.

**Reply by David-Else** [reply]
Afraid that does not work. The directory is full of directories like `jsonls` etc... I don't know how node looks for global packages normally. (for the node ones, I could symlink the binaries)

**Comment by archseer** [comment, marked as answer]
You could use a custom config that overrides the location for each LSP

```
[[language]]
name = "typescript"
language-server.command = "full/path/here"
```
